game/src/ecs/components/PlayableComponent.java

```diff
@@ -1,11 +1,14 @@
 package ecs.components;
 
 import ecs.entities.Entity;
+import java.util.logging.Logger;
+import logging.CustomLogLevel;
 
 /** Marks an entity as player */
 public class PlayableComponent extends Component {
 
     private boolean playable;
+    private final Logger playableCompLogger = Logger.getLogger(this.getClass().getSimpleName());
```

Comment:
Hier geht dir die Vererbung des Loggers verloren, wenn du einfach nur getSimpleName nimmst. Dies machst du aber nicht konsistent z.b. beim MissingComponent nutzt du ```  Logger.getLogger(this.getClass().getName()) ``` während hier einfach nur ``` Logger.getLogger(this.getClass().getSimpleName()) ```.

Da wir die Klassennamen bisher nur einmal verwenden, ist dies kein Problem. Dennoch sollte entweder immer der komplette Name genommen werden oder immer der ohne simple.

Reply:
Habe es an allen Stellen nun so geändert, dass einheitlich .getName() verwendet wird.
fixed in 52f4671d701138239da1e0e6cc93a3e66b8bd1a8